util/package.go

```diff
@@ -67,20 +67,23 @@ type Package struct {
 
 	// Local path to the package dir
 	BasePath string `json:"-" yaml:"-"`
+
+	ExpandDataStream bool `config:"expand_data_stream,omitempty" json:"expand_data_stream,omitempty" yaml:"expand_data_stream,omitempty"`
 }
 
 // BasePackage is used for the output of the package info in the /search endpoint
 type BasePackage struct {
-	Name        string  `config:"name" json:"name"`
-	Title       *string `config:"title,omitempty" json:"title,omitempty" yaml:"title,omitempty"`
-	Version     string  `config:"version" json:"version"`
-	Release     string  `config:"release,omitempty" json:"release,omitempty"`
-	Description string  `config:"description" json:"description"`
-	Type        string  `config:"type" json:"type"`
-	Download    string  `json:"download" yaml:"download,omitempty"`
-	Path        string  `json:"path" yaml:"path,omitempty"`
-	Icons       []Image `config:"icons,omitempty" json:"icons,omitempty" yaml:"icons,omitempty"`
-	Internal    bool    `config:"internal,omitempty" json:"internal,omitempty" yaml:"internal,omitempty"`
+	Name            string            `config:"name" json:"name"`
+	Title           *string           `config:"title,omitempty" json:"title,omitempty" yaml:"title,omitempty"`
+	Version         string            `config:"version" json:"version"`
+	Release         string            `config:"release,omitempty" json:"release,omitempty"`
+	Description     string            `config:"description" json:"description"`
+	Type            string            `config:"type" json:"type"`
+	Download        string            `json:"download" yaml:"download,omitempty"`
+	Path            string            `json:"path" yaml:"path,omitempty"`
+	Icons           []Image           `config:"icons,omitempty" json:"icons,omitempty" yaml:"icons,omitempty"`
+	Internal        bool              `config:"internal,omitempty" json:"internal,omitempty" yaml:"internal,omitempty"`
+	BaseDataStreams []*BaseDataStream `config:"base_data_streams,omitempty" json:"base_data_streams,omitempty" yaml:"base_data_streams,omitempty"`
```

Comment:
`BaseDataStream` contains redundant data that you can find in package. I think we should keep them only in package to prevent the response body from increasing.

BTW base data streams look to me more like data stream previews or sth (just looking for a different name :)